extras/check_code_style.sh

```diff
@@ -76,7 +76,7 @@ print_usage() {
 Usage: ./extras/check_code_style.sh [options ...]
 
 This script processes C/C++ and Python source code files to verify compliance with the NEST
-coding  style  guidelines.  The  checks are performed the same way as in the NEST Travis CI
+coding  style  guidelines.  The  checks are performed the same way as in the NEST Github Actions CI
```

Comment:
Let's make our life easier the next time we change CI systems ;-)

```suggestion
coding  style  guidelines.  The  checks are performed the same way as in the NEST CI
```